src/app/services/config.service.ts

```diff
@@ -206,7 +215,7 @@ export class ConfigService {
   }
 
   readConfig(): void {
-    this.config = JSON.parse(fs.readFileSync(this.configPath, 'utf8'));
+    this.config = { ...this.getDefaultConfig(), ...JSON.parse(fs.readFileSync(this.configPath, 'utf8')) };
```

Comment:
I dont believe this can work, but maybe I am wrong :)
```ts
this.config = Object.assign({}, this.getDefaultConfig(), JSON.parse(fs.readFileSync(this.configPath, 'utf8')));
```